modules/iam-read-only-policy/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "policy_object" {
+  description = "Policy object if created"
+  value       = one(aws_iam_policy.policy[*])
```

Comment:
There is no need to use 0.15 features here.

Also, it is rather a bad idea to expose the whole resource with all the properties (same as in programming in most cases). Please change to export attributes individually.

Reply:
could argue with that in this particular case but alright will do as you suggested